docker_config/dockerfiles/Dockerfile.worker

```diff
@@ -62,7 +62,7 @@ RUN mkdir ${WORKDIR}/scripts
 
 # Install dependencies
 COPY requirements.txt requirements.txt
-RUN python3.6 -m pip install --user --upgrade pip==20.3.4; \
+RUN python3.6 -m pip install --user --upgrade pip; \
```

Comment:
fyi @jzwang43 this reverts your change here https://github.com/codalab/codalab-worksheets/pull/3986